sdk/test-utils/recorder/src/recorder.ts

```diff
@@ -80,10 +80,15 @@ export function record(testContext: Mocha.Context): Recorder {
     recorder = new NockRecorder(testHierarchy, testTitle);
   }
 
-  // If neither recording nor playback is enabled, requests hit the live-service and no recordings are generated
   if (isRecordMode()) {
+    // If TEST_MODE=record, invokes the recorder, hits the live-service,
+    // saves the recording after the test finishes.
     recorder.record();
+  } else if (isLiveMode()) {
```

Comment:
do we need this section? Since there is no early return and presumably `isPlaybackMode()` will return `false`, it seems like this block can be removed without side effects

Reply:
It is not required, I was just playing around with the if conditional checks and left this part as it is.. since it felt like it gives more clarity to the other Devs.

Removed now.. and kept the comment about the live testing. - 8fc475f

Reply:
It is not required, I was just playing around with the if conditional checks and left this part as it is.. since it felt like it gives more clarity to the other Devs.

Removed now.. and kept the comment about the live testing. - 8fc475f